app/src/main/java/ru/netology/fmhandroid/error/AppError.kt

```diff
@@ -0,0 +1,20 @@
+package ru.netology.fmhandroid.error
+
+import android.database.SQLException
+import java.io.IOException
+
+sealed class AppError(var code: String) : RuntimeException() {
```

Comment:
```suggestion
sealed class AppException(var code: String) : RuntimeException() {
```
Обычно всё, что можно выбросить через `throw` именуют `Exception`. `Error` же можно безопасно вернуть через `return` из функции в качестве показателя неудачного выполнения функции, но нельзя выбросить через `throw`

P.S,: К наследникам тоже относится 😊

Reply:
Исправлено.